repos/system_upgrade/el7toel8/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/do-upgrade.sh

```diff
@@ -17,6 +17,149 @@ NSPAWN_OPTS="--capability=all --bind=/sys --bind=/dev --bind=/dev/pts --bind=/ru
 [ -d /dev/mapper ] && NSPAWN_OPTS="$NSPAWN_OPTS --bind=/dev/mapper"
 export NSPAWN_OPTS="$NSPAWN_OPTS --bind=/run/udev --keep-unit --register=no --timezone=off --resolv-conf=off"
 
+#
+# Temp for collecting and preparing tarbal
+#
+LEAPP_DEBUG_TMP="/tmp/leapp-debug-root"
+
+#
+# Number of times to emit all chunks
+#
+# To avoid spammy parts of console log, second and later emissions
+# take longer delay in-between.  For example, with N being 3,
+# first emission is done immediately, second after 10s, and the
+# third one after 20s.
+#
+IBDMP_ITER=3
+
+#
+# Size of one chunk payload
+#
+IBDMB_CHUNKSIZE=40
```

Comment:
The whole archive will be dumped to the console `IBDMP_ITER` times? 

As for `IBDMB_CHUNKSIZE`:
1. Shouldn't it be `IBDMP_CHUNKSIZE`?
2. It's size in bytes / characters?

Reply:
> The whole archive will be dumped to the console `IBDMP_ITER` times?
> 
> As for `IBDMB_CHUNKSIZE`:
> 
> 1. Shouldn't it be `IBDMP_CHUNKSIZE`?
Good catch
> 2. It's size in bytes / characters?
Since the dump is supposed to be BASE64 and the data is supposed to be UTF-8, the set of characters would cause size in bytes == number of characters

Reply:
@AloisMahdal you should probably check the script by shellcheck

Reply:
yeah, it's number of b64-enoced chars per chunk, which by definition will be ascii, ie. bytes.

I will clarify it in the docstring.

FWIW, this is not directly about the console size footprint (although it does affect how much meta-data there will be) but more about "how big a string can i safely print without something messing with it".  (Ie. the first value is roughly half of the "standard" 80-wide terminal.)

Reply:
> @AloisMahdal you should probably check the script by shellcheck

I have (and the code *I*'m adding is shellcheck-ok), but the typo would not be caught by ShellCheck (it was everywhere).

Reply:
Anyway, I've fixed the typo 827e66e1 and clarified the docstring 5b4c85c